test/linalg/triangular.jl

```diff
@@ -134,14 +144,16 @@ for elty1 in (Float32, Float64, Complex64, Complex128, BigFloat, Int)
         # Unary operations
         @test full(-A1) == -full(A1)
 
-        # copy
-        B = similar(A1)
-        copy!(B,A1)
-        @test B == A1
-        B = similar(A1.')
-        copy!(B, A1.')
-        @test B == A1.'
-
+        # copy and copy! (test views as well, see issue #14317)
+        let vrange = 1:n-1, viewA1 = t1(sub(A1.data, vrange, vrange))
+            # copy
+            @test copy(A1) == copy(full(A1))
+            @test copy(viewA1) == copy(full(viewA1))
+            # copy!
+            B = similar(A1); copy!(B, A1); @test B == A1
```

Comment:
Insofar as I can tell those lines exercise `copy!` the same way but additionally exercise `transpose`, so I dropped them in favor of the explicit, dedicated exercise of `transpose` in the earlier section. If I'm missing something please let me know. In any case I would be happy to toss them back in if you like. Thanks again!